packages/webdriverio/tests/commands/element/isClickable.test.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * @jest-environment jsdom
```

Comment:
Any reason why we need this? We don't have it for other tests either.

Reply:
Copy paste, removing